code/include/Whisker.h

```diff
@@ -128,16 +125,12 @@ class Whisker
 	btScalar calc_volume(btScalar length, btScalar R, btScalar r) const;
 	btScalar calc_damping(btScalar k, btScalar mass, btScalar CoM, btScalar zeta, btScalar dt) const;
 	btScalar calc_stiffness(btScalar E, btScalar I, btScalar length) const;
-	float get_dzeta(int index) const;
-	float get_dphi(int index) const;
 
 public:
-
-	Whisker(btDiscreteDynamicsWorld* world, GUIHelperInterface* m_guiHelper,btAlignedObjectArray<btCollisionShape*>* shapes, std::string w_name, Parameters* parameters);
+	Whisker(GUIHelperInterface* helper, btAlignedObjectArray< btCollisionShape* >* shapes, std::string w_name, Parameters& parameters, btTransform head2origin);
 	~Whisker(){}
+	void initPhysics(btDiscreteDynamicsWorld* world, btRigidBody* head);
```

Comment:
Ideally, initPhysics should be removed from the whisker class, right? As it's setting up parameters for the entire simulation, not only the whisker.

Reply:
nevermind, you use this function to get the world parameters